### ISSUE TITLE
Add integratedTerminal profile configuration to debugging configs

### ISSUE THREAD

**Issue by Shuna322** [microsoft/vscode#132451]
To stay more organized I suggest to add some customization to debugging terminals.
My workflow in VS Code mostly rely on workspace with two projects, back-end and front-end. Inside each project I have different launch options: [Launch Django application, Compound of: [Launch Celery Worker, Launch Celery Beat] and Launch Node React]

I want to achieve automated result that can be found on screenshot here: [Imgur](https://imgur.com/GikAqeS)

Basically I want to start debugging and integratedTerminal's should use selected profile\settings for my every debugging configurations, so I can easily differentiate them.

Example of proposed configuration

`launch.json`
```json
{
    "name": "Celery [Back]",
    "type": "python",
    "request": "launch",
    "console": "integratedTerminal",
    "integratedTerminalProfile": "celery",
    "module": "celery",
}
```

**Comment by weinand** [comment]
Sorry, but there are no plans to support this.
